hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -445,6 +445,11 @@ void notifyBackupComplete() {
 
         // We are the lucky ones since we just managed to complete the last backup for this invocation and since the
         // pendingResponse is set, we can set it on the future.
+        complete(pendingResponse);
+    }
+
+    private void complete(Object pendingResponse) {
```

Comment:
I'm going to leave it as response since that is the primary naming approach within the invocation. Check responses package.